Packs/WALLIXBastion/Integrations/WAB/WAB.py

```diff
@@ -11,34 +11,61 @@ class AuthError(Exception):
     pass
 
 
-def str_arg(args, name):
-    return str(args.get(name, ""))
+Unset = object()
 
 
-def int_arg(args, name):
-    return arg_to_number(args.get(name, None), arg_name=name)
+def str_arg(args: Dict[str, Any], name: str, nullable=False):
+    arg = args.get(name, "")
+    if arg == "":
+        return Unset
+    if nullable and arg == "null":
+        return None
+    return str(arg)
 
 
-def bool_arg(args, name):
-    return argToBoolean(args.get(name, False))
+def int_arg(args: Dict[str, Any], name: str, nullable=False):
+    arg = args.get(name, "")
+    if arg == "":
+        return Unset
+    if nullable and arg == "null":
+        return None
+    return arg_to_number(arg, arg_name=name)
 
 
-def list_arg(args, name):
-    return argToList(args.get(name, []))
+def bool_arg(args: Dict[str, Any], name: str, nullable=False):
+    arg = args.get(name, "")
+    if arg == "":
+        return Unset
+    if nullable and arg == "null":
+        return None
+    return argToBoolean(arg)
+
+
+def list_arg(args: Dict[str, Any], name: str, nullable=False):
+    arg = args.get(name, "")
+    if arg == "":
+        return Unset
+    if nullable and arg == "null":
+        return None
+    return argToList(arg)
 
 
 def add_key_to_outputs(outputs: dict, key_name: str, key_val):
     if type(outputs) is dict and key_name not in outputs:
         outputs[key_name] = str(key_val)
 
 
-def to_markdown(name, t):
+def to_markdown(name: str, t):
     try:
         return tableToMarkdown(name, t)
     except Exception as e:
         return "Success (failed to format output: %s)" % str(e)
 
 
+def build_params(**kwargs):
```

Comment:
why do we need this wrapper function? it only calls to assign_params?

Reply:
Its purpose is just to automate calls to assign_params with the proper arguments, especially values_to_ignore.

Reply:
The best practice is to call the assign_params directly.
Would appreciate if yu rethink about it.
Thank you!